cost-model/src/cost_tracker.rs

```diff
@@ -88,10 +90,18 @@ impl Default for CostTracker {
             secp256k1_instruction_signature_count: 0,
             ed25519_instruction_signature_count: 0,
             in_flight_transaction_count: 0,
+            transaction_cost_histogram: Histogram::new(),
```

Comment:
I think it would be nice to track both estimated and actual costs in separate histos